src/components/User.vue

```diff
@@ -0,0 +1,242 @@
+<!--
+  - @copyright Copyright (c) 2022 Carl Schwan <carl@carlschwan.eu>
+  -
+  - @author Carl Schwan <carl@carlschwan.eu>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+<template>
+	<div class="guest-box">
+		<form action="" method="post">
+			<input type="hidden" name="requesttoken" :value="requesttoken">
+			<fieldset>
+				<div v-if="message !== ''" class="notecard error">
+					{{ message }}
+				</div>
+				<p v-else>
+					{{ t('registration', 'Welcome, you can create your account below.') }}
+				</p>
+
+				<div v-if="additionalHint" class="notecard success">
+					{{ additionalHint }}
+				</div>
+
+				<p v-if="!emailIsOptional || email.length > 0" class="input">
+					<input id="email"
+						type="email"
+						class="input__field"
+						name="email"
+						:value="email"
+						disabled>
+					<label for="email" class="infield">{{ t('registration', 'Email') }}></label>
+					<img id="email-icon"
+						class="input__icon"
+						:src="emailIconPath"
+						alt="">
```

Comment:
Same issues here, needs to use `vue-material-design-icons`
https://github.com/nextcloud/notifications/blob/master/package.json#L31